src/components/FormSignUp.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+	<div
+		class="pt:mt-0 mx-auto flex w-full flex-col items-center justify-center px-6 pt-8 dark:bg-gray-900"
+	>
+		<a
+			:href="url()"
+			class="mb-8 flex items-center justify-center text-2xl font-semibold dark:text-white lg:mb-10"
+		>
+			<img :src="asset('images/logo.svg')" alt="Logo" class="mr-4 h-11" />
+			<span>{{ SITE_TITLE_SHORT }}</span>
+		</a>
+		<!-- Card -->
+		<div class="w-full max-w-xl space-y-8 rounded-lg bg-white p-6 shadow dark:bg-gray-800 sm:p-8">
+			<h2 class="text-2xl font-bold text-gray-900 dark:text-white">Create a Free Account</h2>
+			<form class="mt-8 space-y-6" @submit.prevent="onSubmit" ref="form">
+				<div>
+					<label for="name" class="mb-2 block text-sm font-medium text-gray-900 dark:text-white"
+						>Name</label
+					>
+					<input
+						v-model="formFields.name"
+						type="text"
+						name="name"
+						id="name"
+						class="block w-full rounded-lg border border-gray-300 bg-gray-50 p-2.5 text-gray-900 focus:border-primary-500 focus:ring-primary-500 dark:border-gray-600 dark:bg-gray-700 dark:text-white dark:placeholder-gray-400 dark:focus:border-primary-500 dark:focus:ring-primary-500 sm:text-sm"
+						placeholder="John Doe"
+						required
+					/>
+				</div>
+				<div>
+					<label for="email" class="mb-2 block text-sm font-medium text-gray-900 dark:text-white"
+						>Your email</label
+					>
+					<input
+						v-model="formFields.email"
+						type="email"
+						name="email"
+						id="email"
+						class="block w-full rounded-lg border border-gray-300 bg-gray-50 p-2.5 text-gray-900 focus:border-primary-500 focus:ring-primary-500 dark:border-gray-600 dark:bg-gray-700 dark:text-white dark:placeholder-gray-400 dark:focus:border-primary-500 dark:focus:ring-primary-500 sm:text-sm"
+						placeholder="name@company.com"
+						required
+					/>
+				</div>
+				<div>
+					<label for="password" class="mb-2 block text-sm font-medium text-gray-900 dark:text-white"
+						>Your password</label
+					>
+					<input
+						ref="password"
+						v-model="formFields.password"
+						type="password"
```

Comment:
Should we have a show hide password thing? Would be cool if we could add it as a directive.